mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -231,4 +241,57 @@ public void searchShouldNotReturnResultsFromOtherUsers() throws Exception {
         assertThat(metaDatas.get(0).getPath()).isEqualTo(MailboxPath.inbox(session));
     }
 
+    @ContractTest
+    public void updateAnnotationsShouldUpdateStoredAnnotation() throws MailboxException {
+        MailboxPath inbox = setUpMailbox();
```

Comment:
the problem with this `setupMailbox()` method is that you it does many different things and when I read a test method, I have no idea what the context is without going to the method. The name is too generic, it has too many responsibilities, it looks common to all tests but in fact it's specific. I really think you should inline the code.